web/src/components/pages/RootPage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { useAuth } from "../../../auth";
+
+const RootPage: React.FC<{ default?: boolean }> = () => {
+  const { isAuthenticated, isLoading, login } = useAuth();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (isAuthenticated) {
+      navigate("/dashboard");
+    }
+  }, [isAuthenticated, navigate]);
+
+  return isLoading ? (
+    <h1>Loading</h1>
+  ) : !isAuthenticated ? (
+    <>
```

Comment:
I know this is boiler code we will replace later, but you don't need <></> here. Change it or don't change it :P